src/components/forms/Label.js

```diff
@@ -13,7 +13,7 @@ export const LabelContent = ({id, disabled = false, isRequired = false, type, ch
       htmlFor={id}
       disabled={disabled}
       className={classNames({
-        'utrecht-form-label--openforms-required': isRequired,
+        'utrecht-form-label--openforms-required': isRequired && requiredFieldsWithAsterisk,
```

Comment:
I expected  this fix to trigger a diff in chromatic.

Reply:
we didn't really have any stories with `validate.required` or the explicit config to not use asterisks for required fields, so it makes sense there's no diff.

Or do you mean markup diffs?